RecoEcal/EgammaCoreTools/src/EcalClusterTools.cc

```diff
@@ -1307,8 +1307,11 @@ float EcalClusterTools::getNormedIX(const DetId& id)
     if( id.subdetId()==EcalEndcap ) ix = EEDetId(id).ix();
     if( id.subdetId()==EcalShashlik ) ix = EKDetId(id).ix();
     if( ix != 0) {
-      int iXNorm  = ix - 50;
-      if(iXNorm<=0) iXNorm--;
+      float iXNorm; 
+      //in shashlik ix and iy run from 1 to 210
+      if( id.subdetId()==EcalShashlik ) iXNorm  = ix - 105.5;
+      else iXNorm = ix - 50.5;
+      //      if(iXNorm<=0) iXNorm--;
```

Comment:
1.) authorities can also make mistakes.. 
    How many events did you use to check this and see no visible impact? And if it really makes no difference, why shall we change it? 

2.) Please don't touch Phase I Endcap even the change is small and invisible. We meant to have the Phase I EE to be as same as possible to the current EE including all the variables for Electron ID, because we want to use this as a reference to check the improvements of Phase II EE(s).